doc/dbus_api.md

```diff
@@ -288,3 +288,42 @@ when the question is answered.
 - Attempt -> unsigned 32-bit integer (r)
   Current attempt to decrypt the device. This value is useful for clients to know if the very same
   question is asked again (i.e., when the provided password did not work).
+
+
+## Status
+
+The main object of a service implements the following interface:
+
+### org.opensuse.DInstaller.Status1
+
+#### Properties
+
+- All -> array(dict(string, variant)) (r)
+  All possible statuses:
+  [
+    {"id" => 0, "label" => "error"},
+    {"id" => 1, "label" => "probing"},
+    {"id" => 2, "label" => "probed"},
+    {"id" => 3, "label" => "installing"},
+    {"id" => 4, "label" => "installed"},
+  ]
```

Comment:
maybe we should also mention when service is busy...so in probing and installing service is blocked ( aka busy ). In other states not...maybe there should be boolean for it?

Reply:
well, even if it won't be blocked, still I think for UI it will be interesting info that service doing some long term job and maybe should display progress in such case?

Reply:
Ok, let's evaluate it as part of https://trello.com/c/7a24Yxog/2987-d-installer-define-the-new-status-handling-workflow.